Analysis/Tasks/PWGDQ/tableReader.cxx

```diff
@@ -465,6 +490,7 @@ struct DQTableReader {
       }
     }
     histNames += "PairsMuonSEPM;PairsMuonSEPP;PairsMuonSEMM;";
+    histNames += "PairsEleMuSEPM;PairsEleMuSEPP;PairsEleMuSEMM;";
```

Comment:
here you have to add one pairs hist class for each electron cut, since multiple cuts are defined. You could just add this a few lines above in the electron cuts loop